components/common/header/index.styles.js

```diff
@@ -107,6 +107,11 @@ export const NavLink = styled.li`
         }
       }
     }
+    &.donate_btn {
+      background: #e91e63;
+      color: white;
+      border-radius: 10px;
```

Comment:
Apply these changes:
```suggestion
	border-radius: 6px;
	font-weight: bold;
	line-height: inherit;
```